lib/options.js

```diff
@@ -123,5 +123,11 @@ module.exports = optionator({
         type: "Boolean",
         default: "false",
         description: "Run config initialization wizard"
+    },
+    {
+        option: "cache",
+        type: "Boolean",
+        default: "false",
+        description: "Use a cache to avoid process files that haven't changed since last run. The cache is saved in the file .eslintcache in the directory where the command is executed"
```

Comment:
The description needs to be much shorter, otherwise it wraps like crazy in the command line. Something like, "Only check changed files." will do.